exchanges/deribit/deribit.go

```diff
@@ -892,9 +892,9 @@ func (d *Deribit) SubmitTransferBetweenSubAccounts(ctx context.Context, ccy curr
 }
 
 // SubmitTransferToSubAccount submits a request to transfer a currency to a subaccount
-func (d *Deribit) SubmitTransferToSubAccount(ctx context.Context, ccy string, amount float64, destinationID int64) (*TransferData, error) {
-	if ccy == "" {
-		return nil, fmt.Errorf("%w '%s'", errInvalidCurrency, ccy)
+func (d *Deribit) SubmitTransferToSubAccount(ctx context.Context, ccy currency.Code, amount float64, destinationID int64) (*TransferData, error) {
+	if ccy.IsEmpty() {
+		return nil, fmt.Errorf("%w '%s'", currency.ErrCurrencyCodeEmpty, ccy.String())
```

Comment:
A whole bunch of these would display as "currency code empty `` " you might as well drop the ccy.String() aand just return `currency.ErrCurrencyCodeEmpty` by itself across this file.